lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -1436,9 +1437,11 @@ Force work.  Forgets unmatching tests, applies a curing statement and move on.
 sub force{
```

Comment:
I understand the Skip, Save and Cancel buttons, but what's the Force button for? I mean, we can't force a square peg in a round hole... If the schema forbids data to be stored, no amount of Force is going to help...?

Reply:
Sometimes, the peg problem is not the shape but the color. See the `no_businesses` test for an example. Either we force the user to go back in SL to define the missing business or we force remove the references to the missings.

Reply:
Ok. I see now. So, `force` applies a query which looses the (non-conforming) data. That's clear now. Thanks for the `no_businesses` reference.